rioxarray/xarray_plugin.py

```diff
@@ -36,6 +36,7 @@ def open_dataset(
         variable=None,
         group=None,
         default_name="band_data",
+        open_kwargs={},
```

Comment:
Since a dictionary is mutable, it is best to instead use None as the default argument and then do something like this later: `open_kwargs = {} if open_kwargs is None else open_kwargs` [ref](https://docs.python-guide.org/writing/gotchas/)

Reply:
This is just a passthrough will low to no change to ever be edited. I use `{}` in those trivial cases for readability, but I can change it.